File: Excercise_Files/02_Python/05_threeNumbersIncrease.py
# simple program that takes three numbers from user, and tells if they increase/decrease/neither

# user input three numbers
num1 = int(input('Input the first whole number: '))
num2 = int(input('Input the second whole number: '))
num3 = int(input('Input the third whole number: '))

# check if decreasing
if num1 > num2 and num2 > num3: # if order decrease
  print('Decreasing order')
# or increasing
elif num1 < num2 and num1 < num3: # if order increase
  print('Increasing order')
# or neither increasing or decreasing
else: # if neither of the above
  print('Neither increasing or decreasing order')

File: Excercise_Files/04_javaScript/05_threeNumbersIncrease.js
// simple program that takes three numbers from user, and tells if they increase/decrease/neither

// user input three numbers
let num1 = Number(prompt('First number: '));
let num2 = Number(prompt('Second number: '));
let num3 = Number(prompt('Third number: '));

// check if decreasing
if (num1 > num2 && num2 > num3){
  console.log('Decreasing order');
}
// or increasing
else if (num1 < num2 && num2 < num3){
  console.log('Increasing order');
}
// or neither increasing or decreasing
else {
  console.log('Neither increasing or decreasing order');
}
